src/lib/dnssd/ActiveResolveAttempts.cpp

```diff
@@ -172,20 +194,25 @@ Optional<ActiveResolveAttempts::ScheduledResolve> ActiveResolveAttempts::NextSch
         if (entry.nextRetryDelay > kMaxRetryDelay)
         {
             ChipLogError(Discovery, "Timeout waiting for mDNS resolution.");
-            entry.peerId.SetNodeId(kUndefinedNodeId);
+            entry.attempt.Clear();
             continue;
         }
 
         entry.queryDueTime = now + entry.nextRetryDelay;
         entry.nextRetryDelay *= 2;
 
-        ScheduledResolve result(entry);
-        entry.firstSend = false;
+        Optional<ScheduledAttempt> attempt = MakeOptional(entry.attempt);
+        ChipLogError(Controller, "sending back attempt %d %d", (int) entry.attempt.attemptType, entry.attempt.firstSend);
```

Comment:
consider some updated text/formatting.  Without context, seeing "sending back attempt 1 0" is not very obvious on what it means. If this is debug only, consider deleting.